src/test/java/HomePageTest.java

```diff
@@ -0,0 +1,89 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+
+public class HomePageTest {
+     WebDriver  driver;
+
+    @BeforeMethod
+    public void setup (){
+        driver = new ChromeDriver();
+        driver.get("https://kwidos.com/");
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    }
+
+
+    @AfterMethod
+    public void tearDown(){
+
+        driver.quit();
+    }
+
+    @Test
+    public void titleTest() throws InterruptedException {
+        //  go to home page https://kwidos.com/
+        // verify title is equal to kwidos.
+
+        String title = driver.getTitle();
+        Assert.assertEquals(title, "Kwidos");
+
+    }
+
+
+    @Test
+    public void LoginWrongCredentialsTest() throws InterruptedException {
+        driver = new ChromeDriver();
+        driver.get("https://kwidos.com/auth/login");
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+
+        //email locator
+        Thread.sleep(4000);
```

Comment:
Please use "new WebDriverWait(...).until(..)" instead of "Thread.sleep()". And please fix it everywhere.